### PR TITLE
Test that getDocumentationLink() gets updated before each production release

### PR DESCRIPTION
**Pull request by labkey-adam**

### CODE REVIEW

File: api/src/org/labkey/api/jsp/JspLoader.java [hunk 1]
```diff
@@ -61,12 +61,6 @@ public void moduleStartupComplete(ServletContext servletContext)
         });
     }
 
-    @Deprecated // Temporary -- keep org.scharp.atlas.peptide.PeptideController building
-    public static HttpJspPage createPage(String ignored, String jspFile)
-    {
-        return createPage(jspFile);
-    }
-
```
Comment (labkey-susanh) [inline api/src/org/labkey/api/jsp/JspLoader.java:69]: This doesn't seem related.  Is it just opportunistic to remove it or will it actually break things?

File: api/src/org/labkey/api/Constants.java [hunk 2]
```diff
@@ -151,5 +154,22 @@ public void testLowestSchemaVersion()
             double expected = Math.round(Year.now().getValue() / 100.0);
             assertEquals("It's time to update Constants.getLowestSchemaVersion()!", expected, lowest, 0.0);
         }
+
+        @Test
+        public void testDocumentationLink()
+        {
+            String releaseVersion = AppProps.getInstance().getReleaseVersion();
+
+            if (!AppProps.UNKNOWN_VERSION.equals(releaseVersion))
```
Comment (labkey-susanh) [inline api/src/org/labkey/api/Constants.java:163]: I don't suppose it would be possible for the version to be "Unknown" for a release build. (I don't really know when it could be "Unknown", realistically.)

**Review by labkey-susanh — APPROVED** [review]
I think this is fine, but I do have a couple of questions.

File: api/src/org/labkey/api/jsp/JspLoader.java (hunk 1 above)
Comment (labkey-adam) [reply api/src/org/labkey/api/jsp/JspLoader.java:69]: Opportunistic. Only caller was in the SCHARP modules, changed a few days ago. And Verify Java Build is happy. But any change could break things...

File: api/src/org/labkey/api/Constants.java (hunk 2 above)
Comment (labkey-adam) [reply api/src/org/labkey/api/Constants.java:163]: We can be fairly sure that core module will have a release version. Only scenario I can think of is if someone modified module.xml to remove the line.
